docs/NOW-spec.md

```diff
@@ -396,50 +123,33 @@ The NOW_STRING16 structure is used to represent variable-length strings of up to
     </thead>
     <tbody>
         <tr>
-            <td colspan="8">len</td>
-            <td colspan="24">str (variable)</td>
+            <td colspan="32">msgSize</td>
         </tr>
         <tr>
-            <td colspan="32">...</td>
+            <td colspan="8">msgClass</td>
+            <td colspan="8">msgType</td>
+            <td colspan="16">msgFlags</td>
         </tr>
     </tbody>
 </table>
 
-**len (1 byte)**: An unsigned 8-bit number containing the string length, excluding the null terminator. The maximum value is 15.
-
-**str (variable)**: The UTF-8 encoded string excluding the null terminator.
-
-##### NOW_STRING32
-
-The NOW_STRING32 structure is used to represent variable-length strings of up to 15 characters that can easily fit within a fixed-size buffer of 32 bytes.
+**msgSize (4 bytes)**: The message size, excluding the header size (8 bytes).
 
-<table class="byte-layout">
-    <thead>
-        <tr>
-            <th>0</th><th>1</th><th>2</th><th>3</th><th>4</th><th>5</th><th>6</th><th>7</th>
-            <th>8</th><th>9</th><th>10</th><th>1</th><th>2</th><th>3</th><th>4</th><th>5</th>
-            <th>6</th><th>7</th><th>8</th><th>9</th><th>20</th><th>1</th><th>2</th><th>3</th>
-            <th>4</th><th>5</th><th>6</th><th>7</th><th>8</th><th>9</th><th>30</th><th>1</th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr>
-            <td colspan="8">len</td>
-            <td colspan="24">str (variable)</td>
-        </tr>
-        <tr>
-            <td colspan="32">...</td>
-        </tr>
-    </tbody>
-</table>
+**msgClass (1 byte)**: The message class.
 
-**len (1 byte)**: An unsigned 8-bit number containing the string length, excluding the null terminator. The maximum value is 31.
+| Flag                            | Meaning              |
+|---------------------------------|----------------------|
+| NOW_NEGOTIATION_MSG_CLASS_ID<br>0x10 | Protocol negotiation class |
```

Comment:
diff decided to break here (NOW_HEADER), but basically, new separate message class was added for the capabilities message (exec caps message was removed in favor of global capabilities)